.github/policies/resourceManagement.yml

```diff
@@ -106,10 +106,10 @@ configuration:
 
             1. [ ] Create and link to this PR and the issue a matching issue in the dotnet/docs repo using the [breaking change documentation template](https://aka.ms/dotnet/docs/new-breaking-change-issue), then remove this `needs-breaking-change-doc-created` label.
 
-            2. [ ] Ask a committer to mail the `.NET Breaking Change Notification` DL.
+            2. [ ] Ask a committer to mail the `.NET SDK Breaking Change Notification` email list.
 
 
-            Tagging @dotnet/compat for awareness of the breaking change.
```

Comment:
are we good no longer tagging them? I think that was true for other tooling repo but wasn't sure if that was also true for the SDK.

Reply:
The team owner let me know that they don't look at "mentions", so using them might create a false sense of security.